src/clj_jwt/bouncy_castle_provider.clj

```diff
@@ -0,0 +1,6 @@
+(ns clj-jwt.bouncy-castle-provider
+  (:import java.security.Security
+           org.bouncycastle.jce.provider.BouncyCastleProvider))
+
+(defonce __install__
+  (Security/addProvider (BouncyCastleProvider.)))
```

Comment:
You told me that this part comes from `pandect`. Could you add a reference to it?